File: Getting-Practice-with-Python-and-Javascript/2- Control Flow Structures and Data Structures/Control Flow Structures/Javascript/Excercises/program-4.js
// 4-Escribir un programa que pida al usuario un número entero y muestre por pantalla si es múltiplo de 43.


userInput = parseInt(prompt('Ingresa un numero entero: '))


if (userInput % 43 == 0) {
    console.log(true)
    document.write(`El numero ${userInput} es multiplo de 43`)
} else {
    console.log(false)
    document.write(`El numero ${userInput} no es multiplo de 43`)
}

File: Getting-Practice-with-Python-and-Javascript/2- Control Flow Structures and Data Structures/Control Flow Structures/Python/Excercises/program-4.py
# 4-Escribir un programa que pida al usuario un número entero y muestre por pantalla si es múltiplo de 43.


userInput = int(input(' Ingresa un numero entero: '))

if userInput % 43 == 0:
    print(True)
    print(f'El numero {userInput} es multiplo de 43')
else: 
    print(False)
    print(f'El numero {userInput} no es multiplo de 43')


